modules/users/lib/users/users.go

```diff
@@ -328,32 +328,20 @@ func (u *Users) GetUser(id string) (*nano.User, error) {
 	if rows.Next() {
 		var user nano.User
 
-		var firstName sql.NullString
-		var lastName sql.NullString
-		var email sql.NullString
-		var sam sql.NullString
-		var windowsPassword sql.NullString
-
 		err = rows.Scan(
 			&user.Id,
-			&firstName,
-			&lastName,
-			&email,
+			&user.FirstName,
+			&user.LastName,
+			&user.Email,
 			&user.IsAdmin,
 			&user.Activated,
-			&sam,
-			&windowsPassword,
+			&user.Sam,
+			&user.WindowsPassword,
 		)
```

Comment:
As we don't have null strings in Pg anymore, we don't need to use `sql.NullString` so we can `Scan` directly to `*String`.
It fairly belongs to this commit.